docs/Contributing.md

```diff
@@ -128,6 +128,26 @@ useful as a comment when reading the code? Or does it only distract?
 Is the explanation independent of the current context and is it likely
 to stand the test of time?
 
+## Write useful tests
+
+New code requires new tests to be written that will exercise that code.  This
```

Comment:
nit: the rest of this doc seems to use a single space after period. I'd have a slight preference for keeping things consistent.